src/components/User-Landing-Page/User-Landing.js

```diff
@@ -53,7 +54,9 @@ const UserLandingPage = () => {
                 <h4 className='connection-name'>You are connected with {connection.name} ({connection.country})</h4>
                 <h4 className='about'>{connection.about}</h4>
                 <div className='button-wrapper'>
-                  <button>Send Postcard</button>
+                  <Link to='/create-postcard'>
+                    <button>Send Postcard</button>
+                  </Link>
                   <button>End Connection</button>
```

Comment:
Maybe we should change the wording for end connection to something like "Go on Hiatus" or "Get a new Buddy" or something?